### ISSUE TITLE
PDEs

### ISSUE THREAD

**Issue by YodaEmbedding** [mabotkin/jungler-tracking#2]
You mentioned the heat equation.... do you mean a "spreading out" effect?

If we wanted to improve predictions, I think simple division right down the middle of the map is a good _feature_ (in the language of machine learning). If a jungler is on his bot side jungle, he's unlikely to appear on the topside very shortly!

**Comment by mabotkin** [comment]
Yes, I do mean a spreading out effect! I was hoping to create a model where the jungler is essentially modeled as "heat" because at one moment where we know where they are for certain, we have lesser and lesser information about the location as the time progresses, similar to how heat dissipates in a closed box.

Your idea for dividing the map down the middle is a good idea, but the model I'm looking for aims to be more accurate than just which side of the map the jungler is on, but also hopefully to estimate where he is most likely to be.  I haven't had much time to develop this project further since the last time I worked on it, but I could look to incoporate some of those ideas into the program when I get a chance.  Thanks!
